client/src/features/visualizer-threejs/hooks/useMouseMove.ts

```diff
@@ -51,7 +51,7 @@ export const useMouseMove = ({
                     originalColorsRef.current.set(instanceId, currentColor);
 
                     // Set the new hovered instance color to red
-                    tangleMeshRef.current.setColorAt(instanceId, color);
+                    tangleMeshRef.current.setColorAt(instanceId, SEARCH_RESULT_COLOR);
```

Comment:
I would recommend creating a new constant (in the constants file) 
`const HOVERED_COLOR = SEARCH_RESULT_COLOR;` so if they change the hovered color we only update one constant